incubator/crate/Chart.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v1
+name: crate
+version: 0.1.0
+description: CrateDB is a distributed SQL database that handles massive amounts of machine data in real-time. This is a Helm chart for CrateDB, to simplify even further its deployment on Kubernetes clusters.
+appVersion: 4.0.3
```

The current stable version is already `4.0.4`.

---

In my opinion it would make sense to set the `appVersion` value to `4.0`, which is the minor version of CrateDB which is definitely compatible with the provided settings. So you can be sure any version between 4.0.0 and 4.0.n is compatible with this Helm chart.

---

That was just to match the image (which was the latest at the time) - I like your idea; I don't think it goes against Helm's 'best practices' which says on it:

> It is a way of specifying the version of the application. For example, the drupal chart may have an appVersion: 8.2.1, indicating that the version of Drupal included in the chart (by default) is 8.2.1.

Taking 'may' per RFC2119 and given that saying 'this includes 4.0 by default' if it includes '4.0.4' is hardly false, I think we can do that.

```suggestion
appVersion: 4.0
```